### DISCUSSION TITLE
Your opinion on October / Winter

### DISCUSSION THREAD

**Discussion by HitArrowLegend** [GinoPane/oc-blogtaxonomy-plugin#90] [General]
Hello, 

I was wondering what were your thought about the Split between October and Winter CMS and how that will impact Taxonomy later on.

Will you release a version on both CMS, will you stick with October, or will you switch to Winter ?

**Comment by GinoPane** [comment]
@HitArrowLegend to be honest this is the first time I heard about it 😄 
What is it? I see that the repository for it does not even exist and the website is just a subscription form. Is it a new, better version of October?

**Reply by HitArrowLegend** [reply]
well, it's not super clear yet as both parties have yet to do a proper announcement about it but some serious stuff happened last week. 

Basically the team split over some internal stuff that we don't know about (yet?). So Winter CMS will branch from October and be its own thing, and apparently, for now, nothing will change as the core will be shared by both CMS. 

You can join Winter CMS' discord and ask questions, Luke and Ben are pretty responsive and helpful about this whole situation

**Reply by GinoPane** [reply]
Ok, what I found out in the October Slack is that this "split" is unwanted decision probably. I don't want to judge contributors and maintainers, so let's see how it goes. I believe if the Winter CMS really appears, guys would like to keep compatibility of the plugins, because otherwise they will lose the community. For plugin creators it means that nothing changes (we would probably need to publish plugins on one more market or something like that).

Here's the comment from October's co-founder: https://github.com/octobercms/october/commit/3e83fbab71adfcf0d4cd3905261faa3ed9b7ff27#commitcomment-47920921

**Reply by GinoPane** [reply]
@HitArrowLegend ok, now the split and reasons are clear. And probably I'll stick to the Winter CMS eventually. 

I don't like the "only-paid" approach chosen for October CMS. I don't earn anything by making plugins but I have to pay in order to develop them 🤯 

I would expect at least development licenses.

**Reply by HitArrowLegend** [reply]
Yeah... that move was unexpected  ... We can't even add free plugins to the v1 of October that they said would remain free... I'm glad you're here :D
